themes/default/content/docs/clouds/aws/get-started/deploy-changes.md

```diff
@@ -382,29 +382,34 @@ var indexHtml = new BucketObject("index.html", BucketObjectArgs.builder()
 {{% choosable language yaml %}}
 
 ```yaml
-ownership-controls:
-  type: aws:s3:BucketOwnershipControls
-  properties:
-    bucket: ${my-bucket.id}
-    rule:
-      objectOwnership: ObjectWriter
 
-public-access-block:
-  type: aws:s3:BucketPublicAccessBlock
-  properties:
-    bucket: ${my-bucket.id}
-    blockPublicAcls: false
+resources:
+  # ...
 
-index.html:
-  type: aws:s3:BucketObject
-  properties:
-    bucket: ${my-bucket.id}
-    source: ${homepage}
-    contentType: text/html
-    acl: public-read
-  options:
-    dependsOn:
-      - ${public-access-block}
+  ownership-controls:
+    type: aws:s3:BucketOwnershipControls
+    properties:
+      bucket: ${my-bucket.id}
+      rule:
+        objectOwnership: ObjectWriter
+
+  public-access-block:
+    type: aws:s3:BucketPublicAccessBlock
+    properties:
+      bucket: ${my-bucket.id}
+      blockPublicAcls: false
+
+  index.html:
+    type: aws:s3:BucketObject
+    properties:
+      bucket: ${my-bucket.id}
+      source:
+        fn::fileAsset: ./index.html
```

Comment:
This is the substantive bit. I went with this approach because it's (a) simpler than adding a separate `variables`block (and updating the content to explain what this is, etc.) and (b) more closely aligned with the other languages, which also declare these `FileAsset`s inline.